delivery_seur/models/seur_request.py

```diff
@@ -252,6 +252,7 @@ def tracking_state_update(self):
             "ENTREGA EFECTUADA": "customer_delivered",
             "ENTREGADO EN PUNTO": "customer_delivered",
             "ENTREGADO CAMBIO SIN RETORNO": "customer_delivered",
+            "No existen expediciones para la búsqueda realizada": "no_update",
         }
         return {
             "tracking_state_history": "\n".join(
```

Comment:
I think that here is where the history gets accumulated each time a new state arrives. Can it be?

Reply:
Bueno, realmente se machaca lo que haya cada vez: https://github.com/OCA/l10n-spain/blob/8c672e53d39d67df139ab04b627f2a922a91ed44/delivery_seur/models/delivery_carrier.py#L176

El caso es que estoy viendo que no se actualizan correctamente expediciones nuevas tampoco. Tengo que revisarlo un poco más a fondo.